trieste/acquisition/function/function.py

```diff
@@ -762,3 +763,74 @@ def acquisition(x: TensorType) -> TensorType:
         return tf.exp(tf.linalg.logdet(covariance + jitter))
 
     return acquisition
+
+
+class BALD(SingleModelAcquisitionBuilder):
+    """
+    Builder for the *Bayesian Active Learning By Disagreement* acquisition function defined in
+    :cite:`houlsby2011bayesian`. The acquisition function computes the information gains of the predictive entropy.
+
+    This acquisition function is intended to use for Binary Gaussian Process Classification model with Bernoulli likelihood.
```

Comment:
perhaps mention that integrating over nuisance parameters is not currently supported (eq 6 from the paper)?